app/config/config_ci.yml

```diff
@@ -86,7 +88,7 @@ parameters:
         to_id: new-subject
         to_name: New Subject
     rds_articles:
-        '12345': https://repro.elifesciences.org/example.html
+        'id-of-article-with-rds': https://repro.elifesciences.org/example.html
```

Comment:
just noticing that the current configuration format is not forward-compatible for when we need to add a download link as an article will then have two link values (HTML page and DAR/whatever archive). Happy to postpone that change if needed.

Reply:
Related to https://github.com/elifesciences/journal/pull/1220/files#r374168992, unclear if the old and new way are meant to co-exist.

Reply:
From https://github.com/elifesciences/issues/issues/5347 my understanding is yes, because `prod` should continue showing the old way until the new way feature flagged in `preview` is ready to be released to the public.

(looking forward to the new new way)

Reply:
Ok; suggest we rename `rds_articles` to `example_rds_articles` or something for the _existing_ implementation, freeing up `rds_articles` for the new way (which can then be changed to supporting HTML and DAR URIs).

Reply:
Agreed to do the free-up in separate PRs on `journal` and `journal-formula`.

Reply:
https://github.com/elifesciences/journal/pull/1223 will rename the existing config. This PR will need to be updated to display that if the feature flag is disabled.